src/wallet/rpcwallet.cpp

```diff
@@ -306,7 +306,7 @@ static UniValue getrawchangeaddress(const JSONRPCRequest& request)
         throw JSONRPCError(RPC_WALLET_ERROR, "Error: This wallet has no available keys");
     }
 
-    OutputType output_type = pwallet->m_default_change_type != OutputType::CHANGE_AUTO ? pwallet->m_default_change_type : pwallet->m_default_address_type;
+    OutputType output_type = pwallet->m_default_change_type ? *pwallet->m_default_change_type : pwallet->m_default_address_type;
```

Comment:
fa078530bc3824d7d5a1cc4f39b617054b08d77e

nit, I think you can use `value_or()` which is available in boost and std.

Reply:
Ah good point